Adding string.h header to deal with memcpy error on Ubuntu-16.04.7 LTS
Author: J. Sassmannshausen / GSTT 
diff --git a/ucx-1.11.2-orig/src/uct/ib/base/ib_device.h b/ucx-1.11.2/src/uct/ib/base/ib_device.h
index ac69544..5eb39b0 100644
--- a/ucx-1.11.2-orig/src/uct/ib/base/ib_device.h
+++ b/ucx-1.11.2/src/uct/ib/base/ib_device.h
@@ -8,6 +8,7 @@
 #ifndef UCT_IB_DEVICE_H
 #define UCT_IB_DEVICE_H
 
+#include <string.h>
 #include "ib_verbs.h"
 
 #include <uct/api/uct.h>
diff --git a/ucx-1.11.2-orig/src/uct/ib/base/ib_verbs.h b/ucx-1.11.2/src/uct/ib/base/ib_verbs.h
index 524139c..4822477 100644
--- a/ucx-1.11.2-orig/src/uct/ib/base/ib_verbs.h
+++ b/ucx-1.11.2/src/uct/ib/base/ib_verbs.h
@@ -19,6 +19,7 @@
 #include <infiniband/verbs_exp.h>
 #endif
 
+#include <string.h>
 #include <errno.h>
 
 #include <ucs/type/status.h>
